app/views/admin/dashboard/index.html.haml

```diff
@@ -6,11 +6,7 @@
 
 - unless @system_checks.empty?
   .flash-message-stack
-    - @system_checks.each do |message|
-      .flash-message{ class: message.critical ? 'alert' : 'warning' }
-        = t("admin.system_checks.#{message.key}.message_html", value: message.value ? content_tag(:strong, message.value) : nil)
-        - if message.action
-          = link_to t("admin.system_checks.#{message.key}.action"), message.action
+    = render @system_checks
```

Comment:
Would be great to find a way to do this to ... the entire rest of the sections on this page.